src/App.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import Header from "./components/Header";
+import CreateArea from "./components/CreateAreaObject";
+import Note from "./components/Note";
+
+import "./App.css";
+
+function App() {
+  const [notes, setNotes] = useState([]);
+
+  function addNote(newNote) {
+    setNotes((prevNotes) => [...prevNotes, newNote]);
+  }
+
+  function deleteNote(id) {
+    setNotes((prevNotes) => prevNotes.filter((note) => note.id !== id));
+  }
+
+  function handleEdit(id, updateNote, handleEdits) {
+    const pushNote = notes.map((note, index) => {
```

Comment:
insert or replace is a better term to describe what you're doing here

push means adding to the end of a data structure, like a stack or array which isn't what your doing here